translation.rst

```diff
@@ -562,16 +562,22 @@ if you're generating translations with specialized programs or teams.
                 ;
             };
 
-.. note::
-
-    You can also store translations in a database; it can be handled by
-    Doctrine through the `Translatable Extension`_ or the `Translatable Behavior`_
-    (PHP 5.4+). For more information, see the documentation for these libraries.
-
-    For any other storage, you need to provide a custom class implementing the
-    :class:`Symfony\\Component\\Translation\\Loader\\LoaderInterface`
-    interface. See the :ref:`dic-tags-translation-loader` tag for more
-    information.
+Translations of Doctrine Entities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unlike the contents of templates, it's not practical to translate the contents
+stored in Doctrine Entities using translation catalogs. Instead, use the
+Doctrine `Translatable Extension`_ or the `Translatable Behavior`_ (PHP 5.4+).
```

Comment:
```suggestion
Doctrine `Translatable Extension`_ or the `Translatable Behavior`_ (Symfony 5.4+).
```

?

Reply:
I used the same contents as before, but your comment made me think. Should we just remove this? It doesn't matter if its PHP 5.4 or Symfony 5.4 ... this is documented in Symfony 5.4 branch and up, so both of those PHP and Symfony requirements will be met.

Reply:
indeed it should (may) be removed as it will be the defacto context

Reply:
OK. Removed. Thanks!